e2e/resources/workbench-config.ts

```diff
@@ -37,6 +38,7 @@ const test = {
   userEmailDomain: '@fake-research-aou.org',
   collaboratorUsername: process.env.TEST_COLLABORATOR || 'puppetmaster@fake-research-aou.org',
   writerUserName: process.env.TEST_WRITER || 'puppetmaster@fake-research-aou.org',
+  readerUserName: process.env.TEST_READER || 'puppetcitester2@fake-research-aou.org',
```

Comment:
Add a reader in every env for `workspace-reader-actions.spec.ts` so the testing is more targeted. Reader testing was part of `workspace-actions-share.spec`.